### DISCUSSION TITLE
Project Demo

### DISCUSSION THREAD

**Discussion by francqz31** [iSiddharth20/Generative-AI-Based-Spatio-Temporal-Fusion#28] [Show and tell]
Hey nice project , do you have any kind of a video demo to judge the proposed quality?

**Comment by iSiddharth20** [comment]
Hello @francqz31 ,

I do have a video comparing results from 4 sets of experiments. 
Since the available GPU resources and training dataset were limited, I was only able to work with image resolution of 270x480 for 24 Image Sequences (180 Frames Each).

In the video you will see results from 4 sets of experiments:
- Method-1 : Baseline : MSE Loss for AutoEncoder | MSE Loss for LSTM
- Method-2 : Composite Loss ‘A’ for AutoEncoder | MSE Loss for LSTM
- Method-3 : MSE Loss for AutoEncoder | Composite Loss ‘B’ for LSTM
- Method-4 : Proposed Methodology : Composite Loss ‘A’ for AutoEncoder  | Composite Loss ‘B’ for LSTM
  - Composite Loss ‘A’ : MSE Loss + 15% MaxEnt Regularization
  - Composite Loss ‘B’ : MSE Loss + 10% SSIM Regularization
 
It was observed that Proposed Methodology had Better Color Reproduction and Reduced Motion Blur. 

Please share your thoughts on the Project and feel free to review [Codebase Explanation](https://github.com/iSiddharth20/Generative-AI-Based-Spatio-Temporal-Fusion/blob/main/CodeExplanation.md) for more details and [Contributing Guidelines](https://github.com/iSiddharth20/Generative-AI-Based-Spatio-Temporal-Fusion/blob/main/CONTRIBUTING.md) if you'd like to contribute to the project.

Feel free to ask any questions you have.

https://github.com/iSiddharth20/Generative-AI-Based-Spatio-Temporal-Fusion/assets/40489787/f67f79f5-30c9-4003-a5da-dea97dec8888
